### ISSUE TITLE
WSL2 and auth registry do not works - can not access to the credentials

### ISSUE THREAD

**Issue by rloutrel** [testcontainers/testcontainers-java#3783]
I am using:
* docker (docker desktop 3.1.0 (build 52484)) 
* in a wsl2 Ubuntu 20.04
* on a Windows 10 Enterprise (Version 20H2 / Build 19042.746)

The registry we are using is a gitlab Registry (requiring authentication)

The docker [login|pull|run] against this registry are working

When I am trying to get an image from this registry, it seems that the credential information can not be found. See failure message bellow:
`
[main] INFO org.testcontainers.utility.RegistryAuthLocator - Failure when attempting to lookup auth config. Please ignore if you don't have images in an authenticated registry. Details: (dockerImageName: testcontainers/ryuk:0.3.0, configFile: /home/myname/.docker/config.json. Falling back to docker-java default behaviour. Exception message: Could not execute [docker-credential-desktop.exe, get]. Error=2, No such file or directory
`

The content of the config.json is as follow:
```json
{
	"auths": {
		"mygitlab.registry.url": {}
	},
	"credsStore": "desktop.exe",
	"experimental": "enabled"
}
```

I assume, that the problem is because testcontainers does not know how to handle the credsStore in this context but I have no clue about how to investigate further.

Does anyone have any hint here?

**Comment by bsideup** [comment]
@rloutrel what version of Testcontainers are you using? Also, do you get the same error when you execute from CLI, as compared to IDE? And, if so, do you use Eclipse as your IDE? It is known for corrupting the `PATH` variable.

**Comment by rloutrel** [comment]
I close the ticket.


For the record:
Testcontainer version was 1.15.0 
The problem was occuring when using Intellij idea runner.

With a standard mvn command, I had the following failure:
```
Status 404: {"message":"No such image: testcontainers/ryuk:0.3.0"}
```

I made a docker pull and the problem was not present anymore.

I deleted the image from my local registry and updated to Testcontainer version 1.15.2 after I saw https://github.com/testcontainers/testcontainers-java/issues/3574. I did not analyse deeper (Version diff, fix due to 3574, ...)
